webapp/app/js/compareperf.js

```diff
@@ -91,7 +91,9 @@ perf.controller('CompareResultsCtrl', [
     function displayResults(rawResultsMap, newRawResultsMap) {
       $scope.compareResults = {};
       $scope.titles = {};
-      window.document.title = "Perfherder Compare Revisions";
+      window.document.title = ($scope.originalRevision + " (" + $scope.originalProject.name + 
```

Comment:
I can't remember what I said in the bug originally, but I think this would read slightly better if we said "Comparison between ... and ..." instead of "... to ..."